src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -100,6 +101,11 @@ public static async Task ProcessLogFileAsync(string filePath, string toolFormat,
             {
                 RaiseLogProcessed(ExceptionalConditions.InvalidJson);
```

Comment:
>RaiseLogProcessed [](start = 16, length = 17)

This function shows message in infobar for all issues related to Sarif log itself e.g. invalid Json / log has empty result / log has configuration error/tool execution error etc.